modules/ui/src/Form/StockTransactions1.php

```diff
@@ -43,14 +43,6 @@ public function buildForm(array $form, FormStateInterface $form_state) {
   /**
    * {@inheritdoc}
    */
-  public function validateForm(array &$form, FormStateInterface $form_state) {
```

Comment:
Thanks @mglaman looks like this is the way to go. @olafkarsten are you happy with this?